web-registry/src/features/ecocredit/CreateBatchBySteps/CreateBatchMultiStepForm/CreditBasics.tsx

```diff
@@ -39,7 +39,7 @@ export interface CreditBasicsFormValues {
   projectId: string;
   startDate: Date | null;
   endDate: Date | null;
-  metadata?: Partial<VCSBatchMetadataLD> | string | undefined;
+  metadata?: Partial<VCSBatchMetadataLD>;
```

Comment:
outside the scope of this, but we dont’ want this typed as `json` or `string` i dont’ think. Should be one or the other

Reply:
agreed. This may have been from our temporary way of handling metadata. I think it's ok to do this now.